static/.htaccess

```diff
@@ -8,519 +8,18 @@ Options -Indexes -MultiViews
-## index pages are not served as /index/ anymore but /index.html
-RewriteRule ^(.*)/index/$ /$1 [R=301,L]
-## tags do not exist anymore, redirect to main page
-RewriteRule ^(.*)/tags/$ / [R=301,L]
-## duplicate sections at the end where removed
-RewriteRule ^(.*)/(.+)/\2/?$ /$1/$2 [R=301,L]
-RewriteRule ^(.*/components/modeler/dmn/)camunda-modeler-dmn/$ /$1 [R=301,L]
-RewriteRule ^(.*/components/modeler/camunda-modeler/element-templates/)camunda-modeler-element-templates/$ /$1 [R=301,L]
-RewriteRule ^(.*/components/modeler/bpmn/)modeler-bpmn/$ /$1 [R=301,L]
-
-# disabled blog, redirect to index page
-RewriteRule ^blog/ / [R=301,L]
-
-# redirect best-practices overview page
-RewriteRule ^docs/components/best-practices/?$ /docs/components/best-practices/overview/ [R=301,L]
-
-# Optimize docs get their own section of the site
-## The initial implementation of optimize section incorrectly placed all components docs in the root of optimize/. 
-##   These rules redirect from there into a components/ folder.
-RewriteRule ^optimize/next/what-is-optimize(\/?)$ /optimize/next/components/what-is-optimize/ [R=301,L]
-RewriteRule ^optimize/next/userguide/(.*)$ /optimize/next/components/userguide/$1 [R=301,L]
-RewriteRule ^optimize/what-is-optimize(\/?)$ /optimize/components/what-is-optimize/ [R=301,L]
-RewriteRule ^optimize/userguide/(.*)$ /optimize/components/userguide/$1 [R=301,L]
-RewriteRule ^optimize/next/components/userguide/processes/ /optimize/next/components/userguide/process-dashboards [R=301,L]
-
-## these rules previously existed, but are now updated to point at the /optimize section:
-RewriteRule ^docs/components/optimize/userguide/combined-reports/(.*)$ /optimize/userguide/combined-process-reports/$1 [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/process-analysis/overview/(.*)$ /optimize/userguide/process-analysis/process-analysis-overview/$1 [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/process-analysis/overview$ /optimize/userguide/process-analysis/process-analysis-overview/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/decision-analysis/overview/(.*)$ /optimize/userguide/decision-analysis/decision-analysis-overview/$1 [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/decision-analysis/overview$ /optimize/userguide/decision-analysis/decision-analysis-overview/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/additional-features/filters/(.*)$ /optimize/userguide/process-analysis/filters/$1 [R=301,L]
-
-## these rules cover all pages that were moved from /docs/components/optimize into /optimize/components:
-RewriteRule ^docs/components/optimize/what-is-optimize(\/?)$ /optimize/components/what-is-optimize/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/(.*)$ /optimize/components/userguide/$1 [R=301,L]
+RewriteRule ^docs/apis-clients/?$ /1.2/docs/apis-tools/working-with-apis-tools/ [R=301,L]
+RewriteRule ^docs/apis-clients(.*)$ /1.2/docs/apis-tools$1 [R=301,L]
```

Comment:
Some of these rewrite rules were adjusted from the originals, because they were targeting pages that don't exist.